modules/manage/pages/security/authorization/acl.adoc

```diff
@@ -1,14 +1,14 @@
-= Configure Authorization
+= Access Control Lists
 :description: ACLs are the main mechanism supported by Redpanda to manage user permissions.
-:page-aliases: features:acls.adoc, security:acls.adoc, security:authorization.adoc
+:page-aliases: features:acls.adoc, security:acls.adoc
 :page-categories: Management, Security
 
 
-Authorization works in tandem with xref:./authentication.adoc[authentication]. Access-control lists (ACLs) provide a way to configure fine-grained access to provisioned users. ACLs work with SASL/SCRAM and with mTLS with principal mapping for authentication.
+Access-control lists (ACLs) provide a way to configure fine-grained access to provisioned users. ACLs work with SASL/SCRAM and with mTLS with principal mapping for authentication.
```

Comment:
unrelated to this PR, but I've noticed it before. ACLs also work with OIDC and GSSAPI principal mapping. Might be worth changing in this round. @deniscoady?

Reply:
@deniscoady - any input on this? Do we need to change anything on the language here?